azure-quantum/tests/unit/test_authentication.py

```diff
@@ -20,6 +20,7 @@
 from azure.identity import (
     CredentialUnavailableError,
     ClientSecretCredential,
+    DefaultAzureCredential,
```

Comment:
Could you please use `_DefaultAzureCredential` that's imported below. This is our implementation based on the original `DefaultAzureCredential`